packages/compass/src/app/index.tsx

```diff
@@ -225,12 +230,22 @@ const Application = View.extend({
                 <RecentQueryStorageProvider
                   value={recentQueryStorageProviderValue}
                 >
-                  <AppRegistryProvider>
-                    <CompassHomePlugin
-                      appName={remote.app.getName()}
-                      getAutoConnectInfo={getAutoConnectInfo}
-                    ></CompassHomePlugin>
-                  </AppRegistryProvider>
+                  <AtlasAuthServiceProvider
+                    value={new CompassAtlasAuthService()}
+                  >
+                    <AtlasAiServiceProvider
+                      defaultHttpHeaders={{
```

Comment:
do we need to pass `defaultHttpHeaders` to each one of the domain services? is there a way to have this setting configured once instead?

Reply:
Yeah, I think Maurizio has a point here, it probably would make sense to have a provider for the http client so that we don't need to proxy its configuration through other services

Reply:
If its fine for both of you, I'll immediately follow up on this PR and add `AtlasHttpProvider` and `AtlasServiceProvider` to clean it up further. Let me know what you think?